audformat/core/utils.py

```diff
@@ -285,8 +285,49 @@ def duration(
     return (ends - starts).sum()
 
 
+def expand_file_path(
+        index: pd.Index,
+        root: str,
+) -> pd.Index:
+    r"""Expand index with relative file path.
```

Comment:
It's not obvious from this line if you expand a relative path or if you expand with a relative path.

Maybe we change to 

```suggestion
    r"""Expand relative path in index with root.
```

I think it's also not completely right, because we just add `root` add the beginning. Which mean if you set `root='/root` and you have already `/data/test.wav` as file path, you will end up with `/root/data/test.wav` which is not really an expansion of a relative file path.